src/components/Sidebar.jsx

```diff
@@ -35,6 +35,13 @@ class Sidebar extends React.Component {
       },
     );
 
+    let plusMinus;
+    if (this.props.textSizeIsLarge) {
+      plusMinus = <span className="sidebar__plusMinus">&#xf010;</span>;
+    } else {
+      plusMinus = <span className="sidebar__plusMinus">&#xf00e;</span>;
```

Comment:
The only thing that’s varying here is the actual escape sequence of the FontAwesome character, why not just store that in the variable? (Or you could maybe even just inline it in the JSX using a conditional)